shotover-proxy/src/transforms/cassandra/sink_cluster/mod.rs

```diff
@@ -187,6 +195,21 @@ fn create_query(messages: &Messages, query: String, version: Version) -> Result<
     })))
 }
 
+//Creating the AST is pretty expensive. Since we are defining the queries
+// ourselves and know they are not batch statements we can skip that check.
+fn create_query(query: String, version: Version, stream_id: i16) -> Message {
+    Message::from_frame(Frame::Cassandra(CassandraFrame {
+        version,
+        stream_id,
+        tracing_id: None,
+        warnings: vec![],
+        operation: CassandraOperation::Query {
+            query: Box::new(CassandraStatement::Unknown(query)),
```

Comment:
To avoid someone accidentally copy/pasting this code into a non sink transform can we add a comment like this just before this line:

```rust
// Using CassandraStatement::Unknown in a non sink transform would cause problems because the query would be inaccessible to transforms further down the chain.
// However using it in a sink transform is a performance hack that is safe because no further transforms need access to the AST.
```